airbyte-config/init/src/main/resources/seed/destination_definitions.yaml

```diff
@@ -167,7 +167,7 @@
 - name: Postgres
   destinationDefinitionId: 25c5221d-dce2-4163-ade9-739ef790f503
   dockerRepository: airbyte/destination-postgres
-  dockerImageTag: 0.3.17
+  dockerImageTag: 0.3.18
```

Comment:
TODO for myself, wait until https://github.com/airbytehq/airbyte/pull/11877 merges and rebase

Reply:
and re bump